hudi-common/src/main/java/org/apache/hudi/common/table/timeline/dto/FileGroupDTO.java

```diff
@@ -46,17 +49,50 @@ public class FileGroupDTO {
   TimelineDTO timeline;
 
   public static FileGroupDTO fromFileGroup(HoodieFileGroup fileGroup) {
+    return fromFileGroup(fileGroup, true);
+  }
+
+  public static List<FileGroupDTO> fromFileGroup(List<HoodieFileGroup> fileGroups) {
```

Comment:
this is a POJO class. lets not add static utils methods here. can we move it to elsewhere. something like DTOUtils may be.